src/descriptors/power_system_inputs.json

```diff
@@ -510,11 +510,11 @@
     "storage": [
         {
             "name": "name",
-            "description": "Gen ID associated with storage"
+            "description": "Storage object name"
         },
         {
-            "name": "name",
-            "description": "Storage object name"
+            "name": "generator_name",
```

Comment:
looks like this will require a change in Tests

Reply:
It requires the companion change in PowerSystemsTestData.  We'll need to merge them to master at the same time.